<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_TCP">
     <property name="title">
      <string>TCP Ethernet</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>88</y>
        <width>88</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Listen on port:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_send_port_TCP">
      <property name="geometry">
       <rect>
        <x>104</x>
        <y>59</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>59</y>
        <width>88</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Send on port:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_listen_port_TCP">
      <property name="geometry">
       <rect>
        <x>104</x>
        <y>88</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ip_TCP">
      <property name="geometry">
       <rect>
        <x>104</x>
        <y>30</y>
        <width>480</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>88</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Send to IP:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonServer">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>60</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Server</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonClient">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>90</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Client</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>96</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="selectBox_serial_port">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Serial Port</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="manufacturerLabel">
        <property name="text">
         <string>Manufacturer:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="serialNumberLabel">
        <property name="text">
         <string>Serial number:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="locationLabel">
        <property name="text">
         <string>Location:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="pidLabel">
        <property name="text">
         <string>Product ID:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="vidLabel">
        <property name="text">
         <string>Vendor ID:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="baudRateLabel">
        <property name="text">
         <string>BaudRate:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="baudRateBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="dataBitsLabel">
        <property name="text">
         <string>Data bits:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="dataBitsBox"/>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="parityLabel">
        <property name="text">
         <string>Parity:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="parityBox"/>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="stopBitsLabel">
        <property name="text">
         <string>Stop bits:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QComboBox" name="stopBitsBox"/>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="flowControlLabel">
        <property name="text">
         <string>Flow control:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QComboBox" name="flowControlBox"/>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="serialPortInfoListBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_ethernet">
     <property name="title">
      <string>UDP Ethernet</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Send on port:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Send to IP:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_ip"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_send_port"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Listen on port:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_listen_port"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>selectBox_serial_port</zorder>
  <zorder>groupBox_ethernet</zorder>
  <zorder>groupBox_TCP</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
